src/core_plugins/timelion/server/series_functions/movingaverage.js

```diff
@@ -9,8 +11,10 @@ module.exports = new Chainable('movingaverage', {
     },
     {
       name: 'window',
-      types: ['number'],
-      help: 'Number of points to average over'
+      types: ['number', 'string'],
+      help: 'Number of points, or a date math expression (eg 1d, 1M) to average over. ' +
+      'If you specify a date math expression, I\'ll get as close as I can given your current interval selection' +
+      'If they don\'t divide nicely things might get weird.'
```

Comment:
Very conversational language, the first person perspective :). I don't think we use this consistently in the help-docs of Timelion (quandl, static are the only ones it seems like), so consider rephrasing.